keyboards/handwired/sick_pad/kb.c

```diff
@@ -0,0 +1 @@
+#include "kb.h"
```

Comment:
```suggestion
#include "sick_pad.h"
```

Please change this header to `#include "sick_pad.h"` and rename this file to "sick_pad.c".